docs/docker/README.md

```diff
@@ -0,0 +1,57 @@
+## TODO for Dockerization
+* Add maintainer(s)
+  - Maintainer for CR Dockerfile needs to be added after pull request is merged. Ideally the maintainer should be someone from official team with appropriate Docker Hub permission to tag image as `box/clusterrunner:latest`. I can volunteer if needed but that will require collaborator permission for Docker Hub.
+* Lean Docker image (consider tiny base e.g. busybox, alpine, etc.)
+  - Currently `Dockerfile.src` uses `python:3.4-slim` as the base. I attempted `python:alpine` as the base but it didn't feel like worth the [trouble](https://github.com/docker/docker/issues/27940). My docker version is `1.12.6` but the problem might have been fixed in `1.13.0+`. I will revisit this in the future.
```

Comment:
@josephharrington The official `python:3.4-slim` [Dockerfile](https://github.com/docker-library/python/blob/dc016eb55d86dd6e23ec5ee2a537ee60ab11c795/3.4/slim/Dockerfile) includes a bunch of dependencies but I am not sure if all of them are needed for CR. I am yet to do a research though...